docs/advanced/ObjectSpreadOperator.md

```diff
@@ -0,0 +1,37 @@
+# Object Spread Operator
+
+Since one of the core tenants of Redux is to never mutate state, you'll often find yourself using `Object.assign` to create
```

Comment:
Nit: let’s refer to it with parens to be consistent with how the rest of the docs refers to methods, i.e. `Object.assign()`. It’s also worth changing the *first* mention of it to be a link to [the relevant MDN page](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign).